documentation/release_notes/release_notes_latest.markdown

```diff
@@ -21,6 +21,30 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 ---
 
+## PuppetDB 6.11.0
+
+### Upgrading
+
+We highly recommend upgrading to PostgreSQL 11 or greater before upgrading to
+PuppetDB 6.11.0. PostgreSQL 11 includes performance improvements which make
+adding a non-null column with a default value much faster and should
+significantly speed up the migration included in this release.
+
+### New features
+ - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date
+
+ - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block
+
+ - #TODO figure out what we want to mention about command broadcast
+
+### Notes
```

Comment:
```suggestion
### Deprecations
```
I feel like people might skip reading the notes

Reply:
👍 updated